python/tskit/cli.py

```diff
@@ -187,15 +188,16 @@ def get_tskit_parser():
         help="Remove any duplicated mutation positions in the source file. ")
     parser.set_defaults(runner=run_upgrade)
 
-    parser = subparsers.add_parser(
+    # suppress fasta visibility pending https://github.com/tskit-dev/tskit/issues/353
+    """parser = subparsers.add_parser(
```

Comment:
Probably better to just comment this out rather than making it a string that does nothing.

Reply:
OK - happy to comment rather than stringify